src/cpprealm/persisted.hpp

```diff
@@ -725,6 +1080,12 @@ std::ostream& operator<< (std::ostream& stream, const persisted<T>& value)
     stream << *value;
 }
 
+template <realm::type_info::Persistable T>
+std::ostream& operator<< (std::ostream& stream, const T& object)
+{
+    return stream;    
```

Comment:
This is incomplete. The easiest way to do this would be do print the JSON dump from the Obj.